ModeratorFrontEnd/AIForOrcas/AIForOrcas.Client.BL/Helpers/DateHelper.cs

```diff
@@ -4,24 +4,22 @@ namespace AIForOrcas.Client.BL.Helpers
 {
 	public static class DateHelper
 	{
-		public static string ToPDT(DateTime datetime, bool timeOnly = false)
+		public static string UTCToPDT(DateTime datetime, bool timeOnly = false)
 		{
-			TimeZoneInfo pstZone = TimeZoneInfo.FindSystemTimeZoneById("Pacific Standard Time");
+			TimeZoneInfo pst = TimeZoneInfo.FindSystemTimeZoneById("Pacific Standard Time");
 			datetime = DateTime.SpecifyKind(datetime, DateTimeKind.Utc);
-			DateTime pstTime = TimeZoneInfo.ConvertTimeFromUtc(datetime, pstZone);
-			var zoneString = pstZone.IsDaylightSavingTime(pstTime) ? "PDT" : "PST";
+			DateTime pstTime = TimeZoneInfo.ConvertTime(datetime, TimeZoneInfo.Utc, pst);
```

Comment:
Out of curiosity: what is the difference between ConvertTimeFromUtc vs ConvertTime used here?

Reply:
Good question to which I don't have an answer. The issue was it was not converting records with daylight saving time. This is the code that was recommended to address it. They looked the same to me but worked differently apparently.